packages/yew/src/html/component/lifecycle.rs

```diff
@@ -219,7 +221,7 @@ pub(crate) struct ComponentState {
     pub(super) render_state: ComponentRenderState,
 
     #[cfg(feature = "csr")]
-    has_rendered: bool,
+    has_rendered: Rc<AtomicBool>,
```

Comment:
`Rc`ing a type that is `Send + Sync` seems weird. Why not use `Cell<bool>` instead of `AtomicBool`